smack-experimental/src/main/java/org/jivesoftware/smackx/mam/MamManager.java

```diff
@@ -401,17 +433,21 @@ public MamQueryResult mostRecentPage(Jid chatJid, int max) throws XMPPErrorExcep
     /**
      * Get the form fields supported by the server.
      * 
+     * @param objectJID The object for which archive is requested
```

Comment:
If `objectJID` means what I think it means, then the javadoc makes no sense. It's the address of the MAM archive, isn't it? And also please s/objectJID/mamArchiveAddress/.